tests/cypress/e2e/siteConfig.cy.ts

```diff
@@ -122,7 +122,7 @@ describe('Rich Text CKeditor 5 - Site level configuration tests', () => {
         ckeditor5.cancel();
     });
 
-    it('Opens link picker when insert a link', function () {
+    it.only('Opens link picker when insert a link', function () {
```

Comment:
Remove `only`. Also I don't know if this is a duplicate of the test you just added, so maybe ok to remove if there's overlap/duplicate. at the very least, this should be moved to link test suite.

Reply:
thank you, good catch; I removed it since it is an exact duplicate of my test in `link.cy.ts`